### PR TITLE
add popup for portfolio cards for pr

### PR DESCRIPTION
**Pull request by supreeti**
Use javascript to Add popup window for each portfolio in mobile and desktop view.
Add close button to visit the page .

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -18,4 +19,285 @@ function close() {
...
+    const popUpBox = document.querySelector('.popup_section');
+    popUpBox.innerHTML = popupEle;
+    const closePopup = document.querySelectorAll('.close_popup');
+    closePopup.forEach((e) => {
+      e.addEventListener('click', () => {
+        popUpContainer.remove();
+      });
+    });
+  });
+});
```
Comment (tresorsawasawa) [inline main.js:303]: - [x] Kindly make sure you update your linters config as currently you are using the wrong one. **[Here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)** is where you can find all information you need about the new linters configuration

File: main.js [hunk 2]
```diff
@@ -18,4 +19,285 @@ function close() {
...
+    description:
+      ['A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required.', 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car. A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.'],
+    image: ['images/Snapshoot Portfolio (3).png', 'images/Snapshoot Portfoliodesk4.png'],
+    technologies: ['html', ' css ', 'javaScript'],
+    technologiesDesk: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    technologiesPop: ['html', ' css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    Source: 'https://github.com/supreeti/PortfolioMobileFirst',
+    liveLink: 'https://supreeti.github.io/PortfolioMobileFirst/',
+  },
+];
```
Comment (tresorsawasawa) [inline main.js]: - [x] Kindly make sure you fix the broken image for the below card to make your application look more professional
Screenshot from 2023-02-10 14-27-23

File: main.js [hunk 3]
```diff
@@ -18,4 +19,285 @@ function close() {
...
+            <footer>
+                <button class="align_3 footer_btn textdecoration" id="see-projects-btn" type="submit">
+                <a href="${cardsData[btnIndex].liveLink}"><span>See Live</span></a>
+                    <img src="images/live.png" alt="">
+                </button>
+
+                <button class="align_3 footer_btn textdecoration" id="see-projects-btn" type="submit">
+                <a href="${cardsData[btnIndex].Source}"><span>See Source</span></a>
+                    <img src="images/github.jpg" alt="">
+                </button>
```
Comment (tresorsawasawa) [inline main.js:280]: - [x] Kindly make sure you remove the underline text decoration on the `See Live` and `See Source` buttons to match your chosen Figma design.
Screenshot from 2023-02-10 14-19-05

File: main.js [hunk 4]
```diff
@@ -18,4 +19,285 @@ function close() {
...
+        </button>
+
+        <button class="footer_btn textdecoration" id="see-projects-btn" type="submit">
+        <a href="${cardsData[btnIndex].Source}"><span>See Source</span></a>
+            <img src="images/github.jpg" alt="">
+        </button>
+        
+    </footer>
+
+</div>
```
Comment (tresorsawasawa) [inline main.js]: - [x] Kindly make sure your popup design sticks as much as possible to the one from the Figma template such as **background color**, **Blur filter effect** **image height**,... Try to check all information from **[Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=oH5WI0fQMPSvteiz-0)** and improve your design
Screenshot from 2023-02-10 14-39-35

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @supreeti 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @supreeti, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**